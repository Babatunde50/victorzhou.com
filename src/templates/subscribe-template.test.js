import React from 'react';
import renderer from 'react-test-renderer';
import SubscriberTemplate from './subscribe-template';

describe('SubscriberTemplate', () => {
  const props = {
    data: {
      site: {
        siteMetadata: {
          title: 'test',
        },
      },
    },
  };

  it('renders correctly', () => {
    const tree = renderer.create(<SubscriberTemplate {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
