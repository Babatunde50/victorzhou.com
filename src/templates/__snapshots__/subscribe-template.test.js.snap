// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubscriberTemplate renders correctly 1`] = `
<StaticQuery
  render={[Function]}
/>
`;
